deploy/cypress_setup.py

```diff
@@ -20,7 +21,32 @@ def cypress(profile: str, schema: str):
     """
     )
 
-    conn.close()
+    start_time = time.time()
+
+    while True:
+        try:
+            # Execute a query to fetch data from the table
+            cur.execute(
+                "SELECT * FROM internal.config where key in ('WAREHOUSE_EVENTS_MAINTENANCE', 'QUERY_HISTORY_MAINTENANCE');"
+            )
+
+            rows = cur.fetchall()
+
+            # if we have two rows, means materialization is complete
+            if len(rows) == 2:
+                break
```

Comment:
It would be nice to print a message that indicates that we've detected successful materialization.